src/state/AttachState.ts

```diff
@@ -84,13 +84,13 @@ export class AttachState implements IState{
         }
 
         await this.attachContainerLogs(CONSENSUS_NODE_LABEL);
-        await this.attachContainerLogs(MIRROR_NODE_LABEL)
+        await this.attachContainerLogs(MIRROR_NODE_LABEL);
         await this.attachContainerLogs(RELAY_LABEL);
 
         let i = 0;
-        while (i++ < Number.MAX_VALUE) {
+        while (i++ < this.loopIterations()) {
           await this.logger.updateStatusBoard();
-          await new Promise((resolve) => setTimeout(resolve, 10000));
+          await new Promise((resolve) => setTimeout(resolve, 1000));
```

Comment:
Quick question... 
Why do we decrease the update interval ? 
We have a 3 sec threshold for the docker container connection check. 
I looked through it and I think we don't have a memory leak ... but at some point (the worse case) we could have 3 docker connection checks running in parallel.

Reply:
Its a mistake on my side, I was playing around with it